tracks/bash/exercises/raindrops/mentoring.md

```diff
@@ -25,6 +25,11 @@ echo "${output:-$1}"
 
 ## Common Suggestions
 
+* Check for a hash-pling: `#!/bin/bash`
+* Consider a more robust hash-pling: `#!/usr/bin/env bash` 
+* Aaron Maxwell's [Bash Strict Mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/): `set -euo pipefail`
```

Comment:
The `set -e` is pretty controversial and has a load of pitfalls. Applying it blindly can be problematic.

Reply:
I removed it and mentioned it under Debug as a possible way to get a minimal working example.